### DISCUSSION TITLE
Sprint 1

### DISCUSSION THREAD

**Discussion by ekhenderson** [cs481-ekh/f22-kitty-cat-studios#77] [Instructor Feedback]
I'm giving you some latitude since you're still figuring out how to integrate the two teams.
- Please don't post discord logs as meeting minutes. That is not an acceptable meeting format. Use Discord voice if you're not meeting in person and post minutes.
- Your retrospective was not very useful since it was a Discord chat
- Individually writing and assigning stories is not how a team works. You should write the stories in the sprint planning meeting together as a team.
- I only see assigned stories in Sprint 1 for the AI team. Where are the General team's assignments?
- You could create another set of sprints in your Project for the General team if that's easier to track.
- All devs should have their assigned stories broken down into tasks and started by now

**Comment by Nathan-Turtleduck** [comment]
Do tasks have to be actual github issues or can they just be items in the checkbox list?

**Reply by ekhenderson** [reply]
The tasks that you spin out from a user story should be github issues so you can track your work on the sprint board (move tasks from To Do to Done, etc.)

If a task is not a work item, e.g. send an email to your sponsor, that does not need a github issue.
